cards/src/main/resources/cards/custom/group6/witch_doctor/minion_entranced_dancer.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "Bladesworn",
+  "name": "Entranced Dancer",
   "baseManaCost": 3,
   "type": "MINION",
   "heroClass": "ROSE",
   "baseAttack": 4,
   "baseHp": 2,
   "rarity": "RARE",
-  "description": "Whenever you target this minion with a spell, gain +1 Health.",
+  "description": "After you cast a spell on this, gain +1 Health.",
   "trigger": {
     "eventTrigger": {
       "class": "TargetAcquisitionTrigger",
```

Comment:
This is still a Whenever instead of an After, maybe it should be an `AfterSpellCastedTrigger`

Reply:
Even on other minions with similar wording, the implementation seems similar to this. For instance, Djinni of Zephyrs, The Voraxx, Vessel of C'thun, and Shapeshifter Ryal.